corona-warn-app/cwa-app-android
# Update dependency group for work and test related (EXPOSUREAPP-3289)

## Relates to
https://jira.itc.sap.com/browse/EXPOSUREAPP-3289

## Description
This PR adresses outdated gradle dependencies, especially worker, test & http related.

## Improvement

- logs showing better background reliability handling with updated worker dependencies
- https://developer.android.com/jetpack/androidx/releases/work#2.4.0
- less failing tests

## Test
- Check if App runs correctly
- Check if Tests run correctly

## Note
Not all dependency groups have been updated. This will be adressed in another PR for better testability.
